brave-core/src/main/java/com/github/kristofa/brave/FlushingSpanCollector.java

```diff
@@ -70,7 +71,14 @@ public void flush() {
   /** Calls flush on a fixed interval */
   static final class Flusher implements Runnable {
     final Flushable flushable;
-    final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+    final ScheduledExecutorService scheduler = Executors.newSingleThreadScheduledExecutor(new ThreadFactory() {
+      @Override
+      public Thread newThread(final Runnable r) {
+        final Thread t = new Thread(r, "brave-core-flushing-span-collector-flusher");
```

Comment:
actually, that's a bad name, too. let's use the concrete class's simple name. One way to do that is to add an ctor param for the name of the thread pool.

```java
new Flusher(this, flushInterval, getClass().getSimpleName());
```

Since this is package protected (aka internal), this change can be safely done.